python/pyspark/ml/regression.py

```diff
@@ -347,6 +347,20 @@ def r2(self):
         """
         return self._call_java("r2")
 
+    @property
+    @since("2.4.0")
+    def r2adj(self):
+        """
+        Returns Adjusted R^2^, the adjusted coefficient of determination.
+
+        .. seealso:: `Wikipedia coefficient of determination \
+        <https://en.wikipedia.org/wiki/Coefficient_of_determination>`
```

Comment:
use the same link from the scaladoc https://en.wikipedia.org/wiki/Coefficient_of_determination#Adjusted_R2

It also doesn't generate properly, you need a trailing `_` I believe (also need to fix `def r2` if you don't mind). 

it should be:
```
.. seealso:: `Wikipedia coefficient of determination \
<http://en.wikipedia.org/wiki/Coefficient_of_determination#Adjusted_R2>`_
```

If you are able to, building the docs to check these is a good idea

Reply:
ok, done.